scripts/apps/authoring-react/authoring-react.tsx

```diff
@@ -1187,24 +1182,16 @@ export class AuthoringReact<T extends IBaseRestApiResponse> extends React.PureCo
         ));
     }
 
-    render() {
-        const state = this.state;
-        const {authoringStorage, fieldsAdapter, storageAdapter, getLanguage, getSidePanel} = this.props;
+    getExposed(): IExposedFromAuthoring<T> | null {
+        const {state} = this;
 
-        if (state.initialized !== true) {
+        if (state.initialized === false) {
```

Comment:
Let's better throw an error, so our return type is not nullable. Returning `null` isn't useful anyway, if I get `null` what I'm supposed to do? call again hoping it would be initialized? :D